app/bundles/CoreBundle/Form/RequestTrait.php

```diff
@@ -109,7 +109,7 @@ protected function prepareParametersFromRequest(Form $form, array &$params, $ent
 
                     // Ensure the value is an array
                     if (!is_array($params[$name])) {
-                        $params[$name] = (false !== strpos($params[$name], '|')) ? explode('|', $params[$name]) : [$params[$name]];
+                        $params[$name] = (false !== strpos($params[$name], '|')) ? explode('|', $params[$name]) : ($params[$name] ? [$params[$name]] : []);
```

Comment:
this could be simplified a little like this:
```suggestion
                        $params[$name] = (false !== strpos($params[$name], '|')) ? explode('|', $params[$name]) : ($params[$name] ?? []);
```